inst/shiny/server.R

```diff
@@ -12,12 +12,14 @@ function(input, output, session) {
     shinyDirChoose(input, "tcr_dir", session = session, roots = volumes)
     shinyFileChoose(input, "reference_fasta", session = session, roots = volumes)
 
-    vals <- reactiveValues(data = .GlobalEnv$.data.object.VDJ, lineage.tab = NULL)
+    vals <- reactiveValues(data = .GlobalEnv$.data.object.VDJ, lineage.tab = NULL, upload_bcr = TRUE, upload_tcr = TRUE, no_vdj = FALSE, sload = FALSE)
     upload <- reactiveValues(
         seurat.rds = NULL,
         bcr.dir = NULL,
         tcr.dir = NULL
     )
+    # Check the upload flags to decide the DALI interface
+    vals$no_vdj <- reactive(if (!vals$upload_tcr & !vals$upload_bcr) TRUE else FALSE)
```

Comment:
if can be removed and something like this should work aswell:
`vals$no_vdj <- reactive(!vald$upload_tcr & !vals$upload_bcr)`